### ISSUE TITLE
Uncaught TypeError: Cannot read property 'position' of undefined

### ISSUE THREAD

**Issue by Arcanemagus** [mehcode/linter-ui-plus#1]
[Enter steps to reproduce:]

1. Load [this file](https://github.com/AtomLinter/linter-jshint/blob/9d6b6408b8501e44de28d6f0dece0401e0643523/CHANGELOG.md) with [linter-proselint](https://github.com/AtomLinter/linter-proselint/releases/tag/v3.2.1) installed (and active).

**Atom**: 1.19.0-beta2 x64
**Electron**: 1.6.9
**OS**: Microsoft Windows 10 Enterprise
**Thrown From**: [linter-ui-plus](https://github.com/mehcode/linter-ui-plus) package 0.1.0

### Stack Trace

Uncaught TypeError: Cannot read property 'position' of undefined

```
At C:\Users\Landon Abney\.atom\packages\linter-ui-plus\lib\components\panel-message.js:55

TypeError: Cannot read property 'position' of undefined
    at PanelMessage.render (/packages/linter-ui-plus/panel-message.js:55:12)
    at Object.initialize (/packages/linter-ui-plus/node_modules/etch/lib/component-helpers.js:32:31)
    at /packages/linter-ui-plus/panel-message.js:20:10)
    at render (/packages/linter-ui-plus/node_modules/etch/lib/render.js:17:25)
    at addChildren (/packages/linter-ui-plus/node_modules/etch/lib/render.js:39:24)
    at render (/packages/linter-ui-plus/node_modules/etch/lib/render.js:29:21)
    at addChildren (/packages/linter-ui-plus/node_modules/etch/lib/render.js:39:24)
    at render (/packages/linter-ui-plus/node_modules/etch/lib/render.js:29:21)
    at Object.initialize (/packages/linter-ui-plus/node_modules/etch/lib/component-helpers.js:40:23)
    at /packages/linter-ui-plus/panel-section.js:26:10)
    at render (/packages/linter-ui-plus/node_modules/etch/lib/render.js:17:25)
    at updateChildren (/packages/linter-ui-plus/node_modules/etch/lib/patch.js:123:34)
    at patch (/packages/linter-ui-plus/node_modules/etch/lib/patch.js:18:9)
    at updateSync (/packages/linter-ui-plus/node_modules/etch/lib/component-helpers.js:108:20)
    at Object.update (/packages/linter-ui-plus/node_modules/etch/lib/component-helpers.js:62:5)
    at Panel.didFocus (/packages/linter-ui-plus/panel.js:136:10)
    at HTMLElement.<anonymous> (~/AppData/Local/atom/app-1.19.0-beta2/resources/app/src/pane-element.js:54:18)
```

### Commands

```
```

### Non-Core Packages

```
atom-autocomplete-php 0.25.6 
atom-beautify 0.30.3 
atom-jade 0.3.0 
atom-material-syntax 1.0.6 
atom-material-syntax-dark 1.0.0 
atom-material-syntax-light 0.4.6 
atom-material-ui 2.0.4 
atom-typescript 11.0.5 
autocomplete-lua 0.8.2 
autocomplete-python 1.9.1 
busy-signal 1.4.3 
docblockr 0.10.5 
editorconfig 2.2.2 
file-watcher 1.1.0 
fizzy 0.21.0 
flow-ide 1.5.0 
highlight-selected 0.13.1 
intentions 1.1.2 
language-ansible 0.2.1 
language-babel 2.66.0 
language-chef 0.10.0 
language-cjson 0.0.1 
language-docker 1.1.8 
language-elixir 0.19.0 
language-erlang 3.2.0 
language-fortran 2.1.0 
language-gettext 0.6.1 
language-glsl 2.0.1 
language-haml 0.24.6 
language-haskell 1.12.1 
language-icinga2 0.3.0 
language-ini 1.19.0 
language-jade 0.7.2 
language-jolie 0.5.0 
language-kotlin 0.5.0 
language-lua 0.9.11 
language-moonscript 1.7.1 
language-postcss 1.3.1 
language-powershell 4.0.0 
language-puppet 0.22.0 
language-r 0.4.1 
language-reg 0.0.0 
language-rust 0.4.11 
language-swift 0.5.0 
language-terraform 0.8.1 
language-vue 0.23.0 
linter 2.2.0 
linter-alex 4.0.0 
linter-clang 4.1.1 
linter-coffeelint 1.2.1 
linter-csslint 1.3.4 
linter-eslint 8.2.1 
linter-flake8 2.2.1 
linter-htmlhint 1.3.3 
linter-js-standard 3.9.1 
linter-js-yaml 1.2.7 
linter-jshint 3.1.5 
linter-jsonlint 1.3.0 
linter-markdown 4.0.1 
linter-perl 0.8.0 
linter-perlcritic 1.4.2 
linter-php 1.3.2 
linter-phpcs 1.6.7 
linter-phpmd 1.5.3 
linter-proselint 3.2.1 
linter-pylama 0.9.3 
linter-pylint 2.1.0 
linter-rails-best-practices 0.2.2 
linter-reek 2.2.1 
linter-rubocop 2.1.1 
linter-ruby 1.2.5 
linter-stylelint 3.10.0 
linter-tidy 2.3.1 
linter-ui-default 1.6.2 
linter-ui-plus 0.1.0 
mapfile-grammar 0.1.0 
merge-conflicts 1.4.5 
minimap 4.28.2 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.6.1 
minimap-linter 2.0.0 
minimap-selection 4.4.0 
monokai 0.24.0 
nord-atom-syntax 0.9.1 
nord-atom-ui 0.11.0 
pigments 0.39.1 
sequential-number 0.5.0 
seti-syntax 1.1.3 
seti-ui 1.9.0 
sort-lines 0.14.0 
Stylus 3.1.1 
svn 0.0.13 
tab-control 0.6.10 
toggle-quotes 1.0.1 
trailing-spaces 0.4.0 
```

**Comment by Arcanemagus** [comment]
Hmmm, looks like it's something specific to the messages of that file.

**Comment by mehcode** [comment]
What linter do you have running for that file?

**Comment by Arcanemagus** [comment]
Looks like it's a combination of [this file](https://github.com/AtomLinter/linter-jshint/blob/9d6b6408b8501e44de28d6f0dece0401e0643523/CHANGELOG.md) and [linter-proselint](https://github.com/AtomLinter/linter-proselint/releases/tag/v3.2.1).

**Comment by Arcanemagus** [comment]
While that breaking file is open, some really _interesting_ things happen:

1. Open "good" file with message
2. Open "broken" file, let error trigger
3. Close good file in background tab
4. See this: 
![image](https://user-images.githubusercontent.com/427137/27802515-0b696860-5fd9-11e7-9028-f91139c6c876.png)
5. Re-opening the "good" file re-associates the name with it:
![image](https://user-images.githubusercontent.com/427137/27802541-321e8f12-5fd9-11e7-9c5d-0bea6e18bb42.png)

----

1. Open "broken" file, let error trigger
2. Open "good" file with message
3. Close "broken" file in background tab
4. See this:
![image](https://user-images.githubusercontent.com/427137/27802577-7b915026-5fd9-11e7-99a3-236759c2b43f.png)


I can file this as a separate issue if you want.

**Comment by mehcode** [comment]
Should be fixed as v0.1.0 — I was not being careful of linter v1 messages.
